From f655e99b92eed996d7eea8486a8023684a207ed7 Mon Sep 17 00:00:00 2001
From: Dominik Poggel <pog@iesy.com>
Date: Mon, 26 Feb 2024 12:20:38 +0100
Subject: [PATCH 13/13] arm64: dts: iesy: disable critical trip point

due to a hardware error the module shuts down sometimes because the internal temp sensor gets faulty values. disable the trip point to prevent the shutdown
---
 arch/arm64/boot/dts/iesy/iesy-imx8mm-osm-sf.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/iesy/iesy-imx8mm-osm-sf.dtsi b/arch/arm64/boot/dts/iesy/iesy-imx8mm-osm-sf.dtsi
index fcb57f3a0019..d9d135253d60 100644
--- a/arch/arm64/boot/dts/iesy/iesy-imx8mm-osm-sf.dtsi
+++ b/arch/arm64/boot/dts/iesy/iesy-imx8mm-osm-sf.dtsi
@@ -70,6 +70,8 @@ reg_usb_b_vbus: regulator@2 {
     };
 };
 
+/delete-node/ &cpu_crit0;
+
 &A53_0 {
 	cpu-supply = <&vdd_arm_0v9>;
 };
-- 
2.30.2

